### PR TITLE
Js week3

### PR DESCRIPTION
**Pull request by AmnaAhmad**
Amna JS-week3

### CODE REVIEW

**Comment by AmnaAhmad** [comment]
Js1-week3

File: Week3/homework/js-exercises/Compliments.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ];
+  //make random compliment
+  const randomCompliment =
+    compliment[Math.floor(compliment.length * Math.random())];
+
+  // (tody Compliment);
+  console.log(`you are ${randomCompliment} ${Amna}`);
+}
+giveCompliment('Amna');
+giveCompliment('Amna');
```
Comment (marcianoviereck) [inline Week3/homework/js-exercises/Compliments.js:25]: nice!

File: Week3/homework/js-exercises/creditCard.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  for (let index in stringOfNumber) {
+    if (stringOfNumber[index] == stringOfNumber[0]) {
+      counter++;
+    }
+  }
+  if (counter == 16) {
+    return true;
+  }
+  return false;}
+  function isGreaterThanSixteen(theNumber) {
```
Comment (marcianoviereck) [inline Week3/homework/js-exercises/creditCard.js:34]: Fancy solution! Nice

File: Week3/homework/js-exercises/shopping.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+let shoppingList = ['Appel', 'pear']
+
+function addToShoppingCart(oneMore){
+    shoppingList.push(oneMore)
+    if(shoppingList.length <=3 ){
+        console.log(shoppingList)
+        return `You bought ${shoppingList}!`}
```
Comment (marcianoviereck) [inline Week3/homework/js-exercises/shopping.js:9]: Don't forget to check the `formatting`. Here the `}` is not in a logical place, but this is for next time:)

**Review by marcianoviereck — APPROVED** [review]
Approved! The homework is looking really good, I would only suggest that you keep checking your `formatting`, because sometimes the `}` were on the wrong line (or in a strange place).

**Pull request closed without merging**